Library/ViewModels/DashboardViewModelTests.swift

```diff
@@ -8,6 +8,12 @@ import XCTest
 @testable import ReactiveExtensions
 @testable import ReactiveExtensions_TestHelpers
 
+typealias ProjectMessageThread = (Project, MessageThread)
+
+private func == (lhs: ProjectMessageThread, rhs: ProjectMessageThread) -> Bool {
+  return lhs.0 == rhs.0 && lhs.1 == rhs.1
```

Comment:
P sure we already have equality operators for tuples for free. Can we delete this?